index.html

```diff
@@ -27,12 +27,21 @@
     <header>
       <a href="#" class="title">Bilal Ahmed</a>
       <nav>
-        <img src="./img/threelines.png" class="menu" alt="menu bar" />
+        <ul class="Navigation">
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About</a></li>
+          <li><a href="#">Contact</a></li>
+          <li><img src="./img/mail.png" alt="mail" /></li>
+        </ul>
       </nav>
+      <img src="./img/threelines.png" class="menu" alt="menu bar" />
     </header>
-    <section>
+    <section class="section-1">
```

Comment:
[OPTIONAL]

- I can see a horizontal scroll bar on the desktop version of your website meaning content is overlapping the website view width which is not a good user experience. Kindly ensure that no horizontal scroll is present on your website. 

![image](https://user-images.githubusercontent.com/63473584/214941842-5e51e966-7634-4fab-8f05-a4cb0a64c982.png)